BUILD.bazel

```diff
@@ -253,7 +253,9 @@ orfs_flow(
         },
         "floorplan": BLOCKS_FLOORPLAN | SRAM_BOOMTILE_FLOOR_PLACE_ARGUMENTS | SKIP_REPORT_METRICS | {
             "CORE_UTILIZATION": "20",
-            "MACRO_PLACE_HALO": "20 20",
```

Comment:
I would like to see an initial diagnosis from @AcKoucher first... but yes. I hope the problem is just some existing rare problem that presents itself with some unfortunate initial conditions and that it can be solved in due course but without urgency.